share/windows/wix-patch.xml

```diff
@@ -5,4 +5,11 @@
               Name="KeePassXC" Icon="ProductIcon.ico"
               WorkingDirectory="INSTALL_ROOT" Advertise="yes" />
   </CPackWiXFragment>
+  <CPackWiXFragment Id="CM_CP_KeePassXC.exe">
+    <ProgId Id="KeePassXC.kdbx" Description="KeePass Password Database" Icon="CM_FP_KeePassXC.exe" IconIndex="1">
+      <Extension Id="kdbx" ContentType="application/x-keepass2">
+        <Verb Id="open" Command="open" TargetFile="CM_FP_KeePassXC.exe" Argument="&quot;%1&quot;"/>
```

Comment:
Why not the colour variant?

Reply:
It didn't look as good, plus the gray one means "locked" which a kdbx file technically is.

Reply:
It is, but IMHO we should be using our official logo.

Reply:
As an alternative, we could use the green logo with the lock overlay. I'll see how that looks.